app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditActivityPresenter.kt

```diff
@@ -121,6 +136,12 @@ class ProfileEditActivityPresenter @Inject constructor(
               }
             }
           )
+      }.setOnDismissListener {
+        profileEditViewModel.isProfileDeletionDialogShown.postValue(false)
       }.create().show()
   }
+
+  private fun getProfileEditViewModelFromFactory(): ProfileEditViewModel {
```

Comment:
update this to `getProfileEditViewModel`

Reply:
We have 
`private val profileEditViewModel`
So I don't think we can name it as getProfileEditViewModel.

Reply:
Have a look at this - https://github.com/oppia/oppia-android/blob/58d94fa6047ebd50d5a721c6516c70db909f7cd9/app/src/main/java/org/oppia/android/app/topic/practice/TopicPracticeFragmentPresenter.kt#L137

Reply:
@FareesHussain Either we have to change the variable name or the function name
Should I use
`private val profileEditVM` as the object or change the function to `getProfileEditVM`

Reply:
Okay will change the variable name to viewModel.

Reply:
> Should I use
> `private val profileEditVM` as the object or change the function to `getProfileEditVM`

You can simply change the fun name to `getProfileEditViewModel`
`getProfileEditViewModelFromFactory` or `getProfileEditVM` does'nt look good